modules/misc/include/materials/Density.h

```diff
@@ -28,6 +28,7 @@ class Density : public Material
   std::vector<const VariableGradient *> _grad_disp;
   const VariableValue & _disp_r;
 
+  std::string _base_name;
```

Comment:
```suggestion
  const std::string _base_name;
```

(I'll make a PR to change the rest to const as well)

Reply:
I agree with adding const and have verified that it does not break anything. Added suggested commit to this PR.

What did you mean by your comment "I'll make a PR to change the rest to const as well?"

Reply:
> What did you mean by your comment "I'll make a PR to change the rest to const as well?"

All classes that use the base name pattern should be consistent. Right now some have this member marked const and some don't. This should be fixed.

Reply:
Right, okay. I can fix those and include them in this PR.